drivers/epd_bw_spi/epd_bw_spi.c

```diff
@@ -0,0 +1,349 @@
+/*
+ * Copyright (C) 2022 Silke Hofstra
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_epd_bw_spi
+ *
+ * @{
+ * @file
+ * @brief       Device driver implementation for the epd_bw_spi display controller
+ *
+ * @author      Silke Hofstra <silke@slxh.eu>
+ * @}
+ */
+#include <string.h>
+#include "byteorder.h"
+#include "epd_bw_spi.h"
+
+#include "epd_bw_spi.h"
+#include "epd_bw_spi_params.h"
+#include "epd_bw_spi_internal.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+void epd_bw_spi_cmd_start(epd_bw_spi_params_t *p, uint8_t cmd, bool cont)
+{
+    DEBUG("[epd_bw_spi] cmd_start: command 0x%02x\n", cmd);
+    if (p->busy_pin != GPIO_UNDEF) {
+        while (gpio_read(p->busy_pin) == p->busy_value) {}
+    }
+    gpio_clear(p->dc_pin);
+    spi_transfer_byte(p->spi, p->cs_pin, cont, (uint8_t)cmd);
+    gpio_set(p->dc_pin);
+}
+
+void epd_bw_spi_write_cmd(epd_bw_spi_params_t *p, uint8_t cmd,
+                          const uint8_t *params, size_t plen)
+{
+    spi_acquire(p->spi, p->cs_pin, SPI_MODE_0, p->spi_clk);
+    epd_bw_spi_cmd_start(p, cmd, plen > 0);
+    if (plen) {
+        spi_transfer_bytes(p->spi, p->cs_pin, false, params, NULL, plen);
+    }
+    spi_release(p->spi);
+}
+
+void epd_bw_spi_read_cmd(epd_bw_spi_params_t *p, uint8_t cmd, uint8_t *params,
+                         size_t plen)
+{
+    spi_acquire(p->spi, p->cs_pin, SPI_MODE_0, p->spi_clk);
+    epd_bw_spi_cmd_start(p, cmd, true);
+    spi_transfer_bytes(p->spi, p->cs_pin, false, NULL, params, plen);
+    spi_release(p->spi);
+}
+
+void epd_bw_spi_wait(epd_bw_spi_params_t *p, uint32_t usec)
```

Comment:
Does this function need to be public (and exported in the `_internal.h` header)? If not please change it to `static`.